long_cat.py

```diff
@@ -0,0 +1,13 @@
+import re
+import string
+
+
+def count_word(text: str)-> dict:
+    """
+    THe function create a dict of words from text and their len
```

Comment:
The function creates a dictionary of words and their lengths from the given text.